boards/FSM/Inc/Io/Io_PrimaryScancon2RMHF.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <stm32f3xx_hal.h>
+#include "Io_Hal.h"
 
 /**
```

Comment:
The Io_Primary/SecondaryScancon2RMHF.h and .c files are not used anymore, please delete